### ISSUE TITLE
[Question] Transparency, Specular Ramp and Normal maps support

### ISSUE THREAD

**Issue by CKCFOGBJ** [Manashiku/MMDGenshin#7]
Hello, I would like to ask a question about your MMDGenshin shader. Recently I noticed that new characters from Genshin (Sumeru ones) use normal maps and something called "Avatar_Tex_Specular_Ramp", also some of the character skins have alpha parts, and I wanted to ask - Will next versions of your shader support those maps and textures?

**Comment by Manashiku** [comment]
yes, this is planned for the next release. There are just a few more things left to do before i can upload it

**Comment by CKCFOGBJ** [comment]
Ooh, okay! Thanks for answering!
